test/host/Coyote/test.cpp

```diff
@@ -1233,10 +1239,29 @@ struct aligned_allocator {
   }
 };
 
+void accl_sa_handler(int)
```

Comment:
this pattern repeats for all the tests. move this boiler-plate code to an additional test-specific utilities file if possible.